CMakeLists.txt

```diff
@@ -42,8 +42,14 @@ if(NOT PYAPR_PREFER_EXTERNAL_LIBAPR OR NOT APR_FOUND)
     set(APR_USE_OPENMP ${PYAPR_USE_OPENMP} CACHE BOOL "" FORCE)
     set(APR_USE_CUDA ${PYAPR_USE_CUDA} CACHE BOOL "" FORCE)
 
+    if(CMAKE_COMPILER_IS_GNUCC)
+        set(APR_PREFER_EXTERNAL_BLOSC OFF CACHE BOOL "" FORCE)
```

Comment:
is there an issue with using external blosc here?

Reply:
Yeah there was, i'll retry it now with master. Not sure why as I was not getting any issues direclty with LibAPR